### ISSUE TITLE
Errors Vers. 1.2.0 with Skada

### ISSUE THREAD

**Issue by Scaver** [spromicky/Details_Covenants#9]
I recievd this errors using it with skada since version 1.2.0 when I logg into WoW:

```
2x Details_Covenants\Details.lua:4: attempt to index local 'detalhes' (a nil value)
[string "@Details_Covenants\Details.lua"]:4: in main chunk

Locals:
_ = "Details_Covenants"
dc = <table> {
 utils = <table> {
 }
 spellMaps = <table> {
 }
 oribos = <table> {
 }
}
oribos = <table> {
 getCovenantIconForPlayer = <function> defined @Details_Covenants\Oribos.lua:111
}
detalhes = nil
(*temporary) = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'detalhes' (a nil value)"
```

and
```
5x Details_Covenants\Core.lua:95: attempt to call method 'replaceDetailsImplmentation' (a nil value)
[string "@Details_Covenants\Core.lua"]:95: in function <Details_Covenants\Core.lua:44>

Locals:
self = DetailsCovenantFrame {
 0 = <userdata>
}
event = "PLAYER_ENTERING_WORLD"
(*temporary) = nil
(*temporary) = <table> {
 utils = <table> {
 }
 spellMaps = <table> {
 }
 replaceSkadaImplmentation = <function> defined @Details_Covenants\Skada.lua:4
 askMessage = "ASK"
 oribos = <table> {
 }
 addonPrefix = "DCOribos"
}
(*temporary) = "attempt to call method 'replaceDetailsImplmentation' (a nil value)"
dc = <table> {
 utils = <table> {
 }
 spellMaps = <table> {
 }
 replaceSkadaImplmentation = <function> defined @Details_Covenants\Skada.lua:4
 askMessage = "ASK"
 oribos = <table> {
 }
 addonPrefix = "DCOribos"
}
registerCombatEvent = <function> defined @Details_Covenants\Core.lua:18
updateGroupRoster = <function> defined @Details_Covenants\Core.lua:26
playerName = "Jelaria"
realmName = "Blackrock"
init = <function> defined @Details_Covenants\Core.lua:32
frame = DetailsCovenantFrame {
 0 = <userdata>
}
```

**Comment by spromicky** [comment]
Thanks for report! Fixed in `1.2.1` version.

**Comment by Scaver** [comment]
Thank you very much. It works :)
